tests/meta-mender-ci/recipes-testing/mender-coverage-client/gobinarycoverage.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "Mender image artifact library"
+GO_IMPORT = "github.com/mendersoftware/gobinarycoverage"
+
+inherit go
+
+SRC_URI = "git://${GO_IMPORT};protocol=https;destsuffix=${PN}-${PV}/src/${GO_IMPORT}"
+SRCREV = "${AUTOREV}"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+
+# S = "${WORKDIR}/git"
+# B = "${WORKDIR}/build"
+
+# GOPATHDIR = "${B}/src/${GO_IMPORT}"
```

Comment:
Remove?

Reply:
Yes, this was simply for testing it out.